src/components/ArticleById.jsx

```diff
@@ -0,0 +1,123 @@
+import { useEffect, useState } from "react";
```

Comment:
Looks good but there is a lot going on in this file. Consider separating out into smaller components

Reply:
Also using fetch in this component, but axios in your utils file. Would be consistent throughout the project